Automattic/woocommerce-payments
# Remove @wordpress/data dependency in the email input iframe file

#### Changes proposed in this Pull Request
@frosso found there are 15 more JS files (~500kB more) added on the shortcode checkout page due to introducing `@wordpress/data` [in the email input iframe file](https://github.com/Automattic/woocommerce-payments/blob/85d18e76cf531f77e47ecf4bbe43b89cc2aae5d1/client/checkout/woopay/email-input-iframe.js#L14).   This PR removes the dependency and check if there's WooPayments on the Blocks or Classic checkout page instead.


#### Testing instructions
**Test with WooCommerce Bookings**
1. Install and activate the WC Bookings plugin
2. Check the "Requires confirmation?" box
3. Use an existing WooPay shopper email on the checkout page
4. The login session iframe should not be opened

**Test Blocks checkout page**
1. As an existing WooPay shopper, add a product to the cart
2. Go to Blocks checkout page and the login session iframe should be opened
3. Go to Classic checkout page and the login session iframe should be opened

-------------------


**Post merge**
